event_registration_partner_unique/exceptions.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, exceptions
+
+
+class UniqueRegistrationPartnerValidationError(exceptions.ValidationError):
+    """Base class for this module's validation errors."""
+
+    def __init__(self, *args, **kwargs):
+        self._args, self._kwargs = args, kwargs
+        value = self._message()
+        super(UniqueRegistrationPartnerValidationError, self).__init__(value)
+
+    def _message(self):
+        """Format the message."""
+        return self.__doc__.format(*self._args, **self._kwargs)
+
+
+class DuplicatedPartnerError(UniqueRegistrationPartnerValidationError):
+    __doc__ = _("Duplicated partners found in event {0}: {1}.")
```

Comment:
Tweak the exception classes so they don't show the full traceback or use those in the core (I think these were used merely for testing purposes back in the day)
![image](https://user-images.githubusercontent.com/5040182/165463269-d426a85b-5200-4e7a-b7c6-d003ee070ad0.png)

Reply:
I don't see the point for these Exceptions.
Why not simply raising `ValidationError` ? 

I would remove them..

And for testing, you can use `self.assertRaisesRegex` to make sure your exception is raised